docs/pages/blog/2021.md

```diff
@@ -0,0 +1,185 @@
+
+What might have been the biggest change in 2021 is the implications of a larger team for the organization structure.
+We have reached an inflection point at around 10 people.
+I could no longer look at all the GitHub issues and PRs.
+At that point, it was requiring >70h/week to do it correctly.
+One individual continuing doing it was not scaling.
+It would prevent growing the team, and more importantly, distract me off the core CEO responsibilities (people, vision, funding).
+So we have broken down the responsibilities
+We have introduced our first layers of management and defined more clearly the purpose of each role.
+
+## Achievements
+
+We have achieved most of what we could have hoped for.
+
+### Company
+
+- The most important, we have welcomed 10 [members](/about/) in the company: Matheus, Michał, Jun, Danilo, Flavien, Benny, Alexandre, Jan, Bharat, Prakhar.
+- We have introduced a [public handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b) on how we run the company.
+  For example, we are using the OKRs work methodology to prioritize our efforts.
+- We held our first company [retreat](/blog/2021-q3-update/#retreat) 🏝 in Lisbon, Portugal 🇵🇹 , for members of the team that were not prevented by COVID-19 related travel restrictions, and who felt safe enough to travel.
+- We have [rebranded](/blog/material-ui-is-now-mui/) the company to a clean 3 letters acronym: mui.com. This move was key for us to grow beyond Material Design.
+
+### MUI Core
+
+- We have released [v5](/blog/mui-core-v5/), 21% of the npm downloads are on this major version now.
+- We have broken down the demos into smaller and focus on one aspect at a time.
+- We have polished and promoted most of the components that were in the lab to `@mui/material`.
+- We have introduced many new components:
+  - [Masonry](/components/masonry/)
+  - [Stack](/components/stack/)
+  - [Trap Focus](/components/trap-focus/)
+  - [Unstyled button](/components/buttons/#unstyled)
+  - [Unstyled slider](/components/slider/#unstyled)
+  - [Unstyled modal](/components/modal/#unstyled)
+  - [Unstyled slider](/components/switches/#unstyled)
+  - [Unstyled tabs](/components/tabs/#unstyled)
+
+### MUI X
+
+- We have released [v5](/blog/mui-x-v5/), 42% of the npm downloads are on this major version now.
+- We have introduced support for [custom locales](/components/data-grid/localization/).
+- We have introduced many new features while consolidating the existing ones:
+  - [Row editing](/components/data-grid/editing/#row-editing)
+  - [Cell editing](/components/data-grid/editing/#cell-editing)
+  - [Column pinning](/components/data-grid/columns/#column-pinning)
+  - [Tree data](/components/data-grid/group-pivot/#tree-data)
+  - [CSV export](/components/data-grid/export/#csv-export)
+  - [Print](/components/data-grid/export/#print)
```

Comment:
```suggestion
  - [Print export](/components/data-grid/export/#print)
```

Reply:
We reference it as only Print in the pricing table comparison. I would argue that Print is clear enough on its own.